cron/worker/main.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/shurcooL/githubv4"
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged